data/css/all-aboard.css

```diff
@@ -81,38 +81,34 @@ header {
     margin: 0 auto;
 }
 
-/* options list styles */
+/* styles for multiple choice sidebars */
 .options {
     margin: 0 0 40px;
     padding: 0;
     list-style: none;
 }
 
-.options dt {
+.options label {
+    display: block;
+    margin-bottom: 20px;
+}
+
+.options label span {
     display: inline-block;
-    margin-right: 5px;
-    padding: 4px 0 0 5px;
-    border: 1px solid #979797;
-    border-radius: 50%;
-    width: 20px;
-    height: 20px;
-    font-size: .75rem;
-    vertical-align: top;
+    margin-left: 10px;
+    max-width: 200px;
+    vertical-align: middle;
 }
 
-.options dt.answer {
+.options .answer {
     background-color: #7ED321;
     color: #333;
+    padding: 10px;
     border-color: #7ED321;
+    border-radius: 4px;
     font-weight: bold;
```

Comment:
I think we should probably revisit the design with Matt. I think the bg color is a bit dark and the font-weight might be a too heavy... but I'm not a designer 🏄